test/loadTest/use_of_epoll/use_epoll.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+"""
+ Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+
+ This file is part of Orion Context Broker.
+
+ Orion Context Broker is free software: you can redistribute it and/or
+ modify it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ Orion Context Broker is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+"""
+__author__ = 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+import sys
+import requests
+import random
+import time
+import datetime
+
+
+class Epoll:
+    """
+    Verify the epoll() instead of select(). See https://jirapdi.tid.es/browse/DM-2375 issue
```

Comment:
First, references to internal pages in public material should be removed.

Second, I'd suggest to change "Verify the epoll() instead of select()" to something more meaningfull, e.g. "Verify that CB works properly with a large number of stablish connections".

This also affects to "internal" parts of the program. E.g. Epoll class name should be changed, used_epoll.py also, etc.